File: chroma-league/src/main/java/com/bonepl/chromaleague/hud/animations/AllyMountainDragonKillAnimation.java
package com.bonepl.chromaleague.hud.animations;

import com.bonepl.chromaleague.rest.eventdata.DragonType;

public class AllyMountainDragonKillAnimation extends StaticBlinkingAnimation {
    public AllyMountainDragonKillAnimation() {
        super(8, DragonType.MOUNTAIN.getColor());
    }
}

File: src/hud/animations/AllyMountainDragonKillAnimation.ts
import {DragonType} from '../../rest/eventdata/DragonType.js';
import {AllyDragonKillBlinkingAnimation} from './AllyDragonKillBlinkingAnimation.js';

export class AllyMountainDragonKillAnimation extends AllyDragonKillBlinkingAnimation {
    constructor() {
        super(DragonType.MOUNTAIN);
    }
}
